src/planscape/planscape/settings.py

```diff
@@ -222,6 +222,8 @@
     "USE_JWT": True,
     "JWT_AUTH_COOKIE": "my-app-auth",
     "JWT_AUTH_REFRESH_COOKIE": "my-refresh-token",
+    "JWT_AUTH_REFRESH_COOKIE_PATH": "/",
+    "JWT_AUTH_SAMESITE": "Lax",
```

Comment:
does this present any risk? not sure what this setting does

Reply:
This is the default we were already using, we just hadn't set it explicitly in our own settings, so it wasn't available to our own code